app/Modules/Group/Events/Traits/IsPublishableApplicationEvent.php

```diff
@@ -18,16 +18,28 @@ public function getEventType(): string
     }
 
     public function getPublishableMessage(): array
-    {
-        return [
-            'expert_panel' => [
-                'id' => $this->group->uuid,
-                'name' => $this->group->displayName,
-                'type' => $this->group->fullType->name,
-                'affiliation_id' => $this->group->expertPanel->affiliation_id
-            ],
-        ];
-    }
+{
+    return [
+        'expert_panel' => array_merge([
+            'id' => $this->group->uuid,
+            'long_name' => $this->group->name,
```

Comment:
As we discussed earlier today, it may be helpful to review the call. I thought the website team was OK with receiving things labelled as either `long_name` or `name`, as long as we are consistent, and there are many other messages where the group name is sent as `name`. Personally, I lean towards `name` since that's also what it is called in the underlying db model.

Reply:
Yes, I understood from the call that other messages use name.  The code is in good enough shape to send to gpm demo and revise later.  There was only one example of name given on the call.